src/containers/tutor-profile/video-presentation/VideoPresentation.jsx

```diff
@@ -2,17 +2,15 @@ import { useTranslation } from 'react-i18next'
 import Box from '@mui/material/Box'
 import Typography from '@mui/material/Typography'
 import VideoBox from '~/components/video-box/VideoBox'
-import videoImg from '~/assets/img/tutor-profile-page/presentationVideoImg.png'
 
-const VideoPresentation = () => {
+const VideoPresentation = (props) => {
```

Comment:
Why did you decide to use `props` and not destructuring them? I also suggest writing this component in typescript. I hope it won't be very difficult.

Reply:
I de-structured the props.
As for the component switching to typscript - this does not apply to my task. a separate ishyu should be described for this one.